bokeh/settings.py

```diff
@@ -43,12 +44,19 @@ def minified(self, default=None):
     def log_level(self, default=None):
         return self._get_str("LOG_LEVEL", default)
 
+    def py_log_level(self, default='info'):
+        level = self._get_str("PY_LOG_LEVEL", default)
+        LEVELS = {'trace': logging.NOTSET,
```

Comment:
Well reading that it seems complicated and confusing, so I'd say remove it just on those grounds.